src/ensembl/src/content/app/about/About.tsx

```diff
@@ -28,7 +28,7 @@ import {
   TopMenu,
   SideMenu
 } from 'src/content/app/about/components/about-menu/AboutMenu';
-import { CircleLoader } from 'src/shared/components/loader/Loader';
+import { CircleLoader } from 'ensemblRoot/src/shared/components/loader';
```

Comment:
😞

Reply:
Argh this again. Problem is it is hard to detect them. I'll remove all ensemblRoot's from this PR now.

Reply:
✅